### PR TITLE
[Merged by Bors] - chore: adaptations for nightly-2024-06-15

### PR DESCRIPTION
**Pull request by kim-em**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Review by grunweg — APPROVED** [review]
Seems straightforward - I'll let another pair of eyes decide. Is `maintainer merge` appropriate here - I would do so if it was.

**Comment by PatrickMassot** [comment]
bors merge

**Pull request closed without merging**
